### PR TITLE
[io] Redirect ux0:app/<title_id> to app0:

### PR DESCRIPTION
**Pull request by d3m3vilurr**
retail vita system allow both `ux0:/...` and `ux0:...`

### CODE REVIEW

File: src/emulator/io/src/io.cpp [hunk 1]
```diff
@@ -170,20 +170,25 @@ SceUID open_file(IOState &io, const std::string &path_, int flags, const char *p
     std::string device_name;
     std::tie(device, device_name) = translate_device(path);
 
-    const std::string ux_app_path{ "ux0:/app/" };
+    const std::vector<std::string> ux_app_paths = { "ux0:/app/", "ux0:app/" };
```
Comment (VelocityRa) [inline src/emulator/io/src/io.cpp]: use `std::array`

Comment (d3m3vilurr) [reply src/emulator/io/src/io.cpp]: done

**Comment by VelocityRa** [comment]
Yeah, ideally we'd 'normalize' the string before doing any redirecting or processing.

Can you do it that way instead, perhaps? Instead of touching redirecting, or any other code, make a `normalize_path` function that removes this `/` beforehand (and in potentially in the future other things too).

**Comment by VelocityRa** [comment]
Discussed in Discord, I'll do this in a different way at some point.

**Pull request closed without merging**